src/main.py

```
@@ -121,9 +121,37 @@ def configure_proxy_fix() -> None:
     ConfigurationError,
 )
 
-# Import constants
-# pylint: disable=import-outside-toplevel, wrong-import-position
-from src.constants import HOLD_TYPES  # noqa: E402 # pylint: disable=E0401
+# Module-level cache for hold types
+_hold_types_cache: Optional[dict[int, str]] = None
```

Pylint (4.0.4)

[convention] Constant name "_hold_types_cache" doesn't conform to UPPER_CASE naming style

(C0103)